webapp/src/hooks/useForm.ts

```diff
@@ -75,12 +75,12 @@ function useForm(initialFormFields: Record<FormFieldNames, ModalFormFieldConfig>
     const isErrorInFormValidation = (): boolean => {
         let errorFields = {};
         Object.keys(initialFormFields).forEach((field) => {
-            if (initialFormFields[field as FormFieldNames].validations) {
-                Object.keys(initialFormFields[field as FormFieldNames].validations ?? '').forEach((validation) => {
+            if (initialFormFields[field as FormFieldNames]?.validations) {
+                Object.keys(initialFormFields[field as FormFieldNames]?.validations ?? '').forEach((validation) => {
```

Comment:
```suggestion
                Object.keys(initialFormFields[field as FormFieldNames].validations).forEach((validation) => {
```
Because it is already being checked in the above if condition.

Reply:
yes but TS is giving error as `Object is possibly 'undefined'`